src/main/java/uk/gov/hmcts/reform/sscs/controller/EvidenceUploadController.java

```diff
@@ -0,0 +1,202 @@
+    }
+
+    private ResponseEntity<Evidence> uploadEvidence(Supplier<Optional<Evidence>> uploadEvidence) {
+        try {
+            Optional<Evidence> evidenceOptional = uploadEvidence.get();
+            return evidenceOptional.map(ResponseEntity::ok)
+                    .orElse(notFound().build());
+        } catch (IllegalFileTypeException exc) {
+            log.info("Cannot upload file illegal file type", exc);
+            return unprocessableEntity().build();
+        }
+    }
+
+    @ApiOperation(value = "List evidence for a hearing",
+            notes = "Lists the evidence that has already been uploaded for a hearing but is still in a draft state."
+    )
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "List of draft evidence")
+    })
+    @GetMapping(
+            value = "{identifier}/evidence",
+            produces = MediaType.APPLICATION_JSON_VALUE
+    )
+    public ResponseEntity<List<Evidence>> listDraftEvidence(
+            @PathVariable("identifier") String identifier
+    ) {
+        return ResponseEntity.ok(evidenceUploadService.listDraftHearingEvidence(identifier));
+    }
+
+    @ApiOperation(value = "Delete COR evidence",
+            notes = "Deletes evidence for a COR appeal. You need to have an appeal in CCD and an online hearing in COH "
+                    + "that references the appeal in CCD."
+    )
+    @ApiResponses(value = {
+            @ApiResponse(code = 204, message = "Evidence deleted "),
+            @ApiResponse(code = 404, message = "No online hearing found with online hearing id"),
+    })
+    @ResponseStatus(value = HttpStatus.NO_CONTENT)
+    @DeleteMapping(
+            value = "{identifier}/evidence/{evidenceId}",
+            produces = MediaType.APPLICATION_JSON_VALUE
+    )
+    public ResponseEntity deleteEvidence(
+            @ApiParam(value = "either the online hearing or CCD case id", example = "xxxxx-xxxx-xxxx-xxxx")@PathVariable("identifier") String identifier,
+            @PathVariable("evidenceId") String evidenceId
+    ) {
+        boolean hearingFound = evidenceUploadService.deleteDraftHearingEvidence(identifier, evidenceId);
+        return hearingFound ? ResponseEntity.noContent().build() : notFound().build();
+    }
+
+    @ApiOperation(value = "Delete COR evidence from a question",
+            notes = "Deletes evidence for a COR appeal. You need to have an appeal in CCD and an online hearing in COH "
+                    + "that references the appeal in CCD."
+    )
+    @ApiResponses(value = {
+            @ApiResponse(code = 204, message = "Evidence deleted "),
+            @ApiResponse(code = 404, message = "No online hearing found with online hearing id"),
+    })
+    @ResponseStatus(value = HttpStatus.NO_CONTENT)
+    @DeleteMapping(
+            value = "{onlineHearingId}/questions/{questionId}/evidence/{evidenceId}",
+            produces = MediaType.APPLICATION_JSON_VALUE
+    )
+    public ResponseEntity deleteEvidence(
```

Comment:
Do we need this endpoint still? If so can we change the API documentation to make it clear what it does as COR is no longer relevant